src/gap.c

```diff
@@ -2632,11 +2651,12 @@ void InitGVarFuncsFromTable (
 
     for ( i = 0;  tab[i].name != 0;  i++ ) {
         UInt gvar = GVarName( tab[i].name );
-        AssGVar( gvar,
-            NewFunction( NameGVarObj( gvar ),
-                         tab[i].nargs,
-                         ArgStringToList( tab[i].args ),
-                         tab[i].handler ) );
+        Obj func = NewFunction( NameGVarObj( gvar ),
+                                tab[i].nargs,
+                                ArgStringToList( tab[i].args ),
+                                tab[i].handler );
+        SetupFuncInfo( func, tab[i].cookie );
+        AssGVar( gvar, func );
         MakeReadOnlyGVar( gvar );
```

Comment:
Note that some kernel extensions in packages do not use `InitGVarFuncsFromTable`.

So I wonder: Will it be safe to use function for which `SetupFuncInfo` has not been invoked? I assume yes, but I wanted to double check.